tests/db/repositories/test_generated_feed_repository_integration.py

```
@@ -0,0 +1,249 @@
+    yield temp_path
+    
+    # Cleanup
+    db.db.DB_PATH = original_path
+    if os.path.exists(temp_path):
+        os.unlink(temp_path)
+
+
+class TestSQLiteGeneratedFeedRepositoryIntegration:
+    """Integration tests using a real database."""
+    
+    def test_create_and_read_generated_feed(self, temp_db):
```

Ruff (0.14.6)

Unused method argument: `temp_db`

(ARG002)